### ISSUE TITLE
The Parsec connected condition fails when switching audio output devices between starting Parsec and AutoGame

### ISSUE THREAD

**Issue by jarrettgilliam** [jarrettgilliam/AutoGame#10]
**Reproduction Steps**

1. Add these options to your parsec config file:
   ```
   server_resolution_x=-1
   server_resolution_y=-1
   server_admin_mute=0
   ```
2. Close AutoGame and Parsec
3. Select an audio output device in Windows
4. Start AutoGame
5. Select a different audio output device
6. Start Parsec
7. Connect over Parsec
8. The game launcher isn't opened

**Expected:**

The launcher is opened regardless of which audio device is selected when starting Parsec and AutoGame.

**Actual:**

The launcher isn't opened depending of which audio device is selected when starting Parsec and AutoGame.

**Comment by jarrettgilliam** [comment]
A real torture test:

1. Add these options to your parsec config file:
   ```
   server_resolution_x=-1
   server_resolution_y=-1
   server_admin_mute=0
   ```
2. Close Parsec and open AutoGame
4. Connect over RDP
5. Start Parsec
6. Disconnect from RDP
7. Connect over Parsec
8. The game launcher isn't opened

**Comment by jarrettgilliam** [comment]
This is mostly resolved with commit 6e7c3631e5e844602fe1a6bde4f8f838bf542486, but not 100%. I no longer think it's possible to always detect Parsec connections by listening for audio events alone.

**Comment by jarrettgilliam** [comment]
Fixed in v2.0.0
